src/ape/managers/networks.py

```diff
@@ -1,16 +1,14 @@
-from typing import ClassVar, Dict, Iterator, Optional
+from typing import Dict, Iterator, Optional
 
 import yaml
-from pluggy import PluginManager  # type: ignore
```

Comment:
I think this was wrong anyways, `ape.plugins import PluginManager` (I wrote another class with same name)